packages/astro-ghostcms/src/integrations/satoriog/astro-ghostcms-satoriog.ts

```diff
@@ -0,0 +1,86 @@
+import { createResolver, defineIntegration } from "astro-integration-kit";
+import { corePlugins } from "astro-integration-kit/plugins";
+import { z } from "astro/zod";
+import c from "picocolors";
+
+export default defineIntegration({
+	name: "@matthiesenxyz/astro-ghostcms-satoriog",
+	optionsSchema: z.object({
+		verbose: z.coerce.boolean().optional(),
+	}),
+	plugins: [...corePlugins],
+	setup({ options }) {
+		const { resolve } = createResolver(import.meta.url);
+
+		return {
+			"astro:config:setup": ({
+				watchIntegration,
+				updateConfig,
+				injectRoute,
+				logger,
+			}) => {
+				watchIntegration(resolve());
+
+				const SatoriLogger = logger.fork(
+					`${c.bold(c.blue("👻 Astro-GhostCMS"))}${c.gray("/")}${c.blue(
```

Comment:
I'd love to see this string as a helper function too

Reply:
Also might be good to have a different name in this package's logs instead of "Astro-GhostCMS". Just then if there's more specific errors for this package the user can pick those up much easier

Reply:
```ts
`${c.bold(c.blue("👻 Astro-GhostCMS"))}${c.gray("/")}${c.blue(
						"SatoriOG",
					)}`,
```

The full snippet actually includes a second name... so it appears in the log as `[Astro-GhostCMS/SatoriOG]`